tests/main/snap-confine-drops-sys-admin/has-sys-admin.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * gcc ./has-sys-admin.c -lcap -o has-sys-admin
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <errno.h>
+#include <sys/capability.h>
+
+bool has_cap(const char *s) {
+	int res;
+	bool rc = false;
+	errno = 0;
+
+	cap_t caps;
+	cap_value_t cap;
+	cap_flag_value_t cap_flags_value;
+
+	if (cap_from_name(s, &cap) < 0) {
+		fprintf(stderr, "could not cap_from_name()\n");
```

Comment:
```suggestion
		fprintf(stderr, "cannot get capability index from capability name\n");
```

Reply:
This falls into the category of https://github.com/snapcore/snapd/pull/7026#discussion_r297842904 where I find the '!= 0' clearer here (since 0 is the only measure of success).